tmva/tmva/inc/TMVA/DNN/Minimizers.h

```diff
@@ -148,8 +152,8 @@ class TGradientDescent
     *  determine if the minimization has converged. */
    bool HasConverged();
    /** Increases the minimization step counter by the test error evaluation
-    * period and uses the provided test error value of to determine if
-    * the minimization has converged. */
+    *  period and uses the provided test error value to determine if the
+    *  minimization has converged. */
    bool HasConverged(Scalar_t testError);
```

Comment:
My change is only a typo fix in the documentation again, but I agree with your comments. I may do a more thorough rewrite if Lorenzo likes your idea too. I didn't want to break the interface for now, since these are public and every user is likely to use it, since Train() and TrainMomentum() are mostly only used for testing.